child_compassion/wizards/global_child_search.py

```diff
@@ -94,6 +94,8 @@ class GlobalChildSearch(models.TransientModel):
     skip = fields.Integer(size=4, default=50000)
     take = fields.Integer(default=80, size=4)
 
+    skip_value = fields.Integer()
```

Comment:
This field is not needed. The context value is something else and doesn't require any field to exist, so you can remove it.